src/xdocs/config_imports.xml

```diff
@@ -818,6 +822,42 @@ import android.*;
           which is part of the Checkstyle distribution.
         </p>
 
+        <h4 id="blacklist-example">Example of blacklist mode</h4>
+        <p>
+          Short description of the behaviour:
+          <ul>
+            <li>Check starts checking from the closest to the current package described inside import control file.</li>
+            <li>If there is matching allow/disallow rule inside the current package then check returns "allowed" or "disallowed" message.</li>
+            <li>If there is no matching allow/disallow rule inside the current package then checks goes up to the parent and continues checking.</li>
```

Comment:
"current package" ==> "current subpackage" 
"up to the parent " ==> "up to the next parent"
"then checks goes up to the parent and continues checking." ==> ", then it continues checking in the parent sub-package."